core/src/main/scala/org/apache/spark/scheduler/CoarseCookSchedulerBackend.scala

```diff
@@ -331,8 +341,10 @@ class CoarseCookSchedulerBackend(
     val cur = System.currentTimeMillis
     if (!ret && cur - lastIsReadyLog > 5000) {
       logInfo(
-        s"Backend is not yet ready. Total acquired executors [$totalExecutorsAcquired] " +
-          s"vs executor limit [$executorLimit]")
+        s"Scheduler backend is not yet ready: " +
+          s"number of acquired executors [$totalExecutorsAcquired] vs " +
```

Comment:
We haven't acquired these. We've only requested them. I think this will be confusing to people. Please change and change the variable while you're at it to  "totalRelevantExecutorRequests" or "outstanding" or something like that.